TAF/testScenarios/functionalTest/V2-API/core-metadata/deviceprofile/PUT-Negative-UploadFile.robot

```diff
@@ -0,0 +1,61 @@
+*** Settings ***
+Resource         TAF/testCaseModules/keywords/commonKeywords.robot
+Suite Setup      Setup Suite
+
+*** Variables ***
+${SUITE}          Core Metadata Device Profile PUT Positive Test Cases
```

Comment:
```suggestion
${SUITE}          Core Metadata Device Profile PUT Negative Test Cases
```
and need a testcase for `Update device profile by upload file with duplicate profile name`